src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1780,12 +1780,8 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         Map<String, List<FeedbackResponseCommentAttributes>> responseComments = new HashMap<String, List<FeedbackResponseCommentAttributes>>();
         
         //Show all questions even if no responses, unless is an ajax request for a specific question.
-        if (role == UserRole.INSTRUCTOR && !params.containsKey(PARAM_QUESTION_ID)) {
-            for (FeedbackQuestionAttributes question : allQuestions) {
-                relevantQuestions.put(question.getId(), question);
-            }
-        }
-        
+        showAllQuestions(role, params, allQuestions, relevantQuestions);
```

Comment:
Additionally, `allQuestions` is of type `List<FeedbackQuestionAttributes>` while `relevantQuestions` is of type `Map<String, FeedbackQuestionAttributes>`. Seems like some renaming is in order.